pkg/util/clusteragent/clusteragent.go

```diff
@@ -59,7 +59,7 @@ func GetClusterAgentClient() (*DCAClient, error) {
 		})
 	}
 	if err := globalClusterAgentClient.initRetry.TriggerRetry(); err != nil {
-		log.Debugf("Cluster Agent init error: %s", err)
+		log.Debugf("Cluster Agent init error: %s", err.Error())
```

Comment:
I think you can just use the` %v` specifier here to avoid calling `err.Error()`

https://twitter.com/davecheney/status/554140326743400448?lang=en